README.md

```diff
@@ -1,13 +1,9 @@
-# Awesome Pyscript
+# PyScript Collective
 
 A list of community contributed content that enriches PyScript's ecosystem.
 
-Learn [how to contribute](CONTRIBUTING.MD) to this repository.
+We strive to make people feel valued and valuable and we hope to make lots of iterations with the help of the community to build a place that's inclusive and uplifiting to all. Please check our [code of conduct](https://github.com/pyscript/governance/blob/main/CODE-OF-CONDUCT.md). If you're interested in contributing head over to our [how to contribute](CONTRIBUTING.MD) document.
 
 ## Index
 
-- Examples
-
-- Video tutorials
-
-- Articles
\ No newline at end of file
+[Examples]()
```

Comment:
+ Video tutorials
+ Articles

being removed? Won't it be nice to point beginners to useful articles and videos?

Reply:
Hey @thenithinbalaji thanks for the comment :)
I'm trying to find what's the best way of showcasing this, tbh.
Since we didn't have any examples for that I decided to remove.
The fact is the repository is not ready yet 😅
I opened it to the public so a colleague could work on it too.
Once I merge this PR that's currently a WIP the repo will be ready to receive contributions and then I'll be really happy to receive a contribution from you! if you have any inputs on how to display this info :)